rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromMockObjectRector/Fixture/some_test_with_mock.php.inc

```diff
@@ -26,7 +26,7 @@ use PHPUnit\Framework\TestCase;
 
 final class SomeTestWithMock extends TestCase
 {
-    public function createMock(): \PHPUnit\Framework\MockObject\MockObject
+    public function test(): \PHPUnit\Framework\MockObject\MockObject
     {
         return $this->createMock('SomeType');
     }
```

Comment:
Here the `createSomeMock()` should be used instead, as a factory method, not a test one.

Reply:
sure https://github.com/rectorphp/rector-src/pull/6171